apps/code-examples/src/app/code-examples/layout/back-to-top/infinite-scroll/demo.component.ts

```diff
@@ -129,21 +129,23 @@ export class DemoComponent implements OnInit {
   ];
 
   public ngOnInit(): void {
-    this.addData(0, 5);
+    this.#addData(0, 5);
   }
 
   public onScrollEnd(): void {
-    this.addData(this.personList.length, 5);
+    this.#addData(this.personList.length, 5);
   }
 
-  private addData(start: number, rowSize: number): void {
+  #addData(start: number, rowSize: number): void {
```

Comment:
Can this be changed to an `async` method with `await` instead of the `.then()` syntax? You can update calls to this function with the `void` keyword to avoid ESLint yelling at you about un-awaited Promises. Same goes for other examples in this PR.

```
void this.#addData(this.personList.length, 5);
```

Reply:
Great tip! Updated.